SimG4Core/Physics/src/G4ProcessTypeEnumerator.cc

```diff
@@ -2,14 +2,14 @@
 
 static const int nprocesses = 49;
 static const std::string g4processes[nprocesses] = {
-    "Primary",         "Transportation", "CoupleTrans",  "CoulombScat", "Ionisation",    "Brems",
-    "PairProdCharged", "Annih",          "AnnihToMuMu",  "AnnihToHad",  "NuclearStopp",  "Msc",
-    "Rayleigh",        "PhotoElectric",  "Compton",      "Conv",        "ConvToMuMu",    "Cerenkov",
-    "Scintillation",   "SynchRad",       "TransRad",     "OpAbsorp",    "OpBoundary",    "OpRayleigh",
-    "OpWLS",           "OpMieHG",        "DNAElastic",   "DNAExcit",    "DNAIonisation", "DNAVibExcit",
```

Comment:
@civanch   Is it on purpose to remove "DNAElastic" and "DNAExcit"?

Reply:
@qliphy , all DNA processes in this list are not used in CMSSW and are not planned to be used - it is low-energy processes in liquid water for studying of low-dose radiation effects on biological molecules. They appears in our list due to historical circumstances. What is important for CMSSW: all other processes should stay with the same indexes and names. So, for us DNA processes are placeholders. In this PR two rare processes for exotica analysis are added instead of two DNA processes. This is safe.